src/seedu/addressbook/AddressBook.java

```diff
@@ -336,6 +345,18 @@ public static String executeCommand(String userInputString) {
     }
 
     /**
+     * Sorts the address book in alphabetical order.
+     * @return feedback display message for the operation result, along with the address book if successful.
+     */
+    private static String executeSortAddressBook() {
+    	if (sortAddressBook()) {    		
+    		executeListAllPersonsInAddressBook();
```

Comment:
are you saving the sorted list separately? Ensure you don't change the actual data in the db or a file unless users explicitly says so. Sort doesn't say so. User only expects a sorted output. However, its good to cache results from a cache so that when someone runs sort again you don't actually have to sort again! This also has some caveats; think about it!